tensorflow/core/platform/profile_utils/cpu_utils.cc

```diff
@@ -53,7 +53,7 @@ static ICpuUtilsHelper* cpu_utils_helper_instance_ = nullptr;
                                        GetCycleCounterFrequency());
 }
 
-/* static */ int64 CpuUtils::GetCycleCounterFrequencyImpl() {
+/* static */ uint64 CpuUtils::GetCycleCounterFrequencyImpl() {
```

Comment:
What is the reason for this change?

Several of the return paths for the function return negative numbers (INVALID_FREQUENCY), some now with implicit casts to unsigned.  

The only use in the rest of this patch adds the opposide cast of INVALID_FREQUENCY back to unsigned.  

What am I missing here?